rootfs/etc/varnish/default.vcl

```diff
@@ -151,11 +143,21 @@ sub vcl_backend_response {
     unset beresp.http.X-Powered-By;
     unset beresp.http.Server;
 
+
     if (beresp.http.Surrogate-Control ~ "ESI/1.0") {
```

Comment:
should we add a comment that this needs to happen before any early return via deliver?

Without testing possibilities it is hard to verify that this stays as it is?

Reply:
Looks not super nice, but maybe worth taking a look at: https://www.varnish-software.com/developers/tutorials/testing-varnish-varnishtest/